### ISSUE TITLE
NullPointerException: com.hbisoft.hbrecorder.ScreenRecordService.onStartCommand

### ISSUE THREAD

**Issue by aruproy** [HBiSoft/HBRecorder#69]
**Describe the bug**
I've received Runtime NullPointerException in production through Google Play console. I don't have access to the device. It would be great if you can resolve this issue. Otherwise, please suggest how I can catch it. Stack trace doesn't show my code.

**Log**
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:4179)
  at android.app.ActivityThread.-wrap21 (Unknown Source)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2119)
  at android.os.Handler.dispatchMessage (Handler.java:108)
  at android.os.Looper.loop (Looper.java:166)
  at android.app.ActivityThread.main (ActivityThread.java:7529)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:245)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:921)
Caused by: java.lang.NullPointerException: 
  at com.hbisoft.hbrecorder.ScreenRecordService.onStartCommand (SourceFile:1)
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:4151)

```
**Can it be reproduced in demo app**
No

**HBRecorder version**
0.1.15

**Device information**
- Huawei Mate 10 lite
- Android 8.0 (SDK 26)

**Screenshots**
Not available.

**Comment by HBiSoft** [comment]
Please try the latest release. You can also add a try/catch when stating the recording.

Let me know how it goes. I will be closing the issue for now, but you are welcome to reopen it if it continues.

**Comment by aruproy** [comment]
@HBiSoft Thank you for your quick response. I released an update with 2.0.0. However I had to quickly rollback to older 0.1.15 due to this error: 

"**HBRecorderOnError: Some settings are not supported by your device. errorCode: 38 reason: 268435556**" 

**I got the error on multiple devices in production:**

CPH1725( OPPO CPH1725) Android 25
X683-OP( INFINIX MOBILITY LIMITED Infinix X683 ) Android 29
CPH1725( OPPO CPH1725 ) Android 25
LenovoTB3-X70L( LENOVO Lenovo TB3-X70L ) Android 23

Hope this information is useful enough to resolve the issue. I'd appreciate your help.
